db_test.go

```diff
@@ -574,6 +574,47 @@ func (s *SurrealDBTestSuite) TestSmartUnMarshalQuery() {
 	})
 }
 
+func (s *SurrealDBTestSuite) TestSmartUnMarshalRaw1Query() {
+	user := testUser{
+		Username: "electwix",
+		Password: "1234",
+	}
+
+	s.Run("raw create query", func() {
+		QueryStr := "Create Only users set Username = $user, Password = $pass"
```

Comment:
maybe we can use full uppercase for SurrealQL operators and statement such as CREATE, ONLY